core/src/summary.cc

```diff
@@ -20,13 +23,13 @@ void Summary::Observe(const double value) {
 }
 
 ClientMetric Summary::Collect() const {
-  auto metric = ClientMetric{};
```

Comment:
Whats wrong with those? We tru to follow the [almost-always-auto](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/) style here.

Reply:
Nothing wrong, it's just simpler and not involving `auto` nor assignment. I have not heard about this style, thanks for the link. In the end, I guess it's just a matter of style, personally I am not a fan of this but no problem to revert it to keep style consistent.

Reply:
Thanks. I'd like to keep those as-is.